### ISSUE TITLE
Switch to database

### ISSUE THREAD

**Issue by athityakumar** [metakgp/naarad-old-data#7]

**Comment by themousepotato** [comment]
As of now, we are dealing with JSON files. So, shifting to MongoDB will be relatively easy.

**Comment by athityakumar** [comment]
Yup, storing them into Mongo collections should be easier. We'll just need to setup MongoDB on our server during deployment.

@themousepotato - Would you like to take this issue up? 😄

**Comment by themousepotato** [comment]
@athityakumar Sure :D . But, it will take time. I haven’t checked out Naarad’s codebase.

**Comment by athityakumar** [comment]
Sure, take your time and ping me / @DefCon-007 if you're stuck somewhere. 😄

**Comment by themousepotato** [comment]
What's the status of [this] issue? Are these somewhat similar?

[this]: https://github.com/metakgp/naarad-source/issues/45

**Comment by athityakumar** [comment]
@themousepotato - Yes, they're actually the same issue. We had naarad and naarad-source as different repos, as one contained just the source code whereas the other had source code + scraped data as JSON files as well. So if we're migrating to Mongo, we can just have one repo that's deployed to Heroku.
